rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -206,18 +212,20 @@ def _decode_withdrawal(
                 )
             ):
                 if (
-                    event.location_label == to and
+                    (event.location_label == to or (event.asset in (A_ETH, A_WETH_SCROLL) and is_weth_user)) and  # noqa: E501
```

Comment:
Shouldn't this have all the possible WETHs? Exactly what we were talking about in the standup. Why only scroll here?

This needs the creation of ETH + WETH group of assets. And then simply the check that `event.asset in group`